api/config/config.example.json

```diff
@@ -15,5 +15,11 @@
   "ledSignIp": "192.168.4.1",
   "GENERAL_API_URL": "http://localhost:8080/",
   "LOGGING_API_URL": "http://localhost:8081/",
-  "MAILER_API_URL": "http://localhost:8082/"
+  "MAILER_API_URL": "http://localhost:8082/",
+  "discordApiKeys": {
+    "CLIENT_SECRET": "XXXXXXXXXXXXXXXXXX",
+    "CLIENT_ID": "XXXXXXXXXXXXXXXXXX",
+    "REDIRECT_URI": "http%3A%2F%2Flocalhost%3A8080%2Fapi%2Fuser%2Fcallback&response_type=code&scope=identify",
```

Comment:
keep in mind that eventually we will no longer redirect users to `localhost` but instead `sce.sjsu.edu`

Reply:
you can keep it as is for now, but we should have checks for when were running in production or not

Reply:
will change this when the site does indeed redirect to sce.sjsu.edu, have to do it in Discord as well